packages/rest-explorer/src/rest-explorer.controller.ts

```diff
@@ -33,12 +33,20 @@ export class ExplorerController {
   }
 
   indexRedirect() {
-    this.response.redirect(301, this.request.url + '/');
+    let url = this.request.url + '/';
+    if (this.request.baseUrl && this.request.baseUrl !== '/') {
+      url = this.request.baseUrl + url;
```

Comment:
FWIW, this can be simplified using [`req.originalUrl`](https://expressjs.com/en/4x/api.html#req.originalUrl), see #2508